content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,160 @@
+
+![Server-Side Batch Changes](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Batch_Changes_Side_by_Side.mp4 "Server-Side Batch Changes")
+
+[Batch Changes]
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+
+##### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+### Get started with Sourcegraph faster than ever
+After many years of building Sourcegraph into a secure and highly-scalable solution, we're ready to bring it to the cloud. Sourcegraph Cloud now offers dedicated, single-tenant instances of Sourcegraph, making it easier than ever for new teams to get started. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+Media: None
+
+Docs: https://docs.sourcegraph.com/cloud
+
+##### 💖 For admins
+
+### Get a deep understanding of usage and measure the value of our platform with new and enhanced in-product analytics
+With 3.42, we introduced an all-new analytics including [XYZ]. Our new analytics make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. We’ve significantly improved our in-product analytics to include the projected value and dev-time-saved for all Sourcegraph features, the ability to export data, individual user usage table, and much more. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data into a user’s preferred systems. We aim to make OpenTelmetry the default way to quickly and easily export observability data from Sourcgraph, starting with tracing in Sourcegraph 4.0. This is a breaking change since we now require customers to export tracing using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), which will now be deployed by default in all Sourcegraph deployment methods, replacing the Jaeger agents and collectors we used to ship by default.
+
+[Admin]
+
+[Docs](https://docs.sourcegraph.com/admin/observability/opentelemetry)
+
+### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+Multi-version upgrades help to reduce the burden and time required to upgrade to the latest version of Sourcegraph. Customers will now be able to seamlessly upgrade from any 3.20+ version to Sourcegraph 4.0 in just a few simple steps. With faster upgrades, customers can quickly start using the new features and enhancements that are released every month. 
+
+![Multi-version upgrades](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Multi-version%20Upgrades.gif "Multi-version upgrades")
```

Comment:
@efritz Hi, I belive you plan to provide another video closer to launch, and when you do, could you please record it as a video and not a gif for quality purposes? Thank you

Reply:
```suggestion
![Multi-version upgrades](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Multi-version%20Upgrades.mp4 "Multi-version upgrades")
```